src/test/java/com/example/bookstore/service/CategoryServiceImplTest.java

```diff
@@ -93,32 +90,25 @@ void findAll_ValidPageable_ReturnsAllCategories() {
     @Test
     @DisplayName("Get category by id")
     void getById_ValidCategoryId_ReturnsCategoryDto() {
-        // Given
         when(categoryRepository.findById(anyLong())).thenReturn(Optional.of(fantasyCategory));
         when(categoryMapper.toDto(any(Category.class))).thenReturn(fantasyCategoryDto);
 
-        // When
         CategoryDto actual = categoryService.getById(CATEGORY_ID);
 
-        // Then
         assertEquals(fantasyCategoryDto, actual);
-        verify(categoryRepository, times(1)).findById(anyLong());
-        verifyNoMoreInteractions(categoryRepository, categoryMapper);
+        verify(categoryRepository).findById(anyLong());
```

Comment:
you did not resolved Andrii's comment about redundant `verify` method calls, please check the documentation of `verify` method to better understand in which cases it is redundant.
<img width="653" alt="image" src="https://github.com/AntonZhdanov/Book-Store/assets/82539553/9b2c5e7b-a684-454a-a36b-c021e4f019e5">